h/htslib/htslib_ubi_8.5.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : htslib
+# Version       : 1.17
+# Source repo   : https://github.com/samtools/htslib
+# Tested on     : UBI 8.5
+# Language      : C
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Vinod K <Vinod.K1@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=htslib
+PACKAGE_VERSION=${1:-1.17}
+PACKAGE_URL=https://github.com/samtools/htslib
+
+OS_NAME=`cat /etc/os-release | grep PRETTY_NAME | cut -d '=' -f2 | tr -d '"'`
+
+yum update -y
```

Comment:
@vinodk99 We need to avoid yum update. Can you please install only required tools or packages and check this